scripts/examples/python/monitor-unwanted-domains.py

```diff
@@ -1,17 +1,37 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
```

Comment:
any reason change this back to `python` other than python3?
I know if no python2 installed, it goes to python3, but if we updated to python3, does't  explicit python3 better?

Reply:
Here is the code snippet for `call_plugin` function. 
```
 31 let call_plugin session_id plugin_name fn_name args =
 32   let plugin_name = find_plugin plugin_name in
 33   (* Marshal the args as XMLRPC *)
 34   let args = List.map (fun (k, v) -> (k, XMLRPC.To.string v)) args in
 35   let call =
 36     XMLRPC.To.methodCall fn_name
 37       [XMLRPC.To.string (Ref.string_of session_id); XMLRPC.To.structure args]
 38   in
 39   let output, _ =
 40     try   
 41       Forkhelpers.execute_command_get_output plugin_name [Xml.to_string call]
 42     with  
 43     | Forkhelpers.Spawn_internal_error (log, output, Unix.WSTOPPED _) ->
 44         raise  
 45         | (Api_errors.Server_error 
 46         |    (Api_errors.xenapi_plugin_failure, ["task stopped"; output; log])
 47         | )

```
Obviously, it does not explicitly specify whether it's using Python 2 or Python 3. This way, it is advisable to explicitly specify the shebang line with python3 to execute these plugins after the py2->py3 upgrade.

P.S. I'm not an OCaml expert, so please correct me if I'm wrong.

Reply:
I also believe `python3` is better here